scruml/assets/scruml.js

```diff
@@ -418,20 +428,38 @@ function removeMember()
         }
 
         divName = divName.substring(3, divName.length - 1);
-        modalAlert("Member " + divName + " has been removed.");
+        // modalAlert("Member " + divName + " has been removed.");
 
         diagram.update(diagram.selectedElement.id());
     });
 }
 
+// ----------
+// Generate random name
+
+function stringGen(len)
```

Comment:
Interesting approaching to adding a placeholder name. There is a minuscule chance that this conflicts with a user inputted name but I think this is safe enough for the time being.